### DISCUSSION TITLE
Multi transitions

### DISCUSSION THREAD

**Discussion by EmilAtanasov** [alexzhornyak/ScxmlEditor-Tutorial#99] [Q&A] [answered]
In page:
https://github.com/alexzhornyak/SCXML-tutorial/blob/master/Tests/ecma/W3C/Mandatory/Auto/report__USCXML_2_0_0___msvc2015_32bit__Win7_1.md
about test 364 wrote Pass. Animation on page:
https://alexzhornyak.github.io/SCXML-tutorial/Doc/state.html#test-364
is correct, but when I open 
https://github.com/alexzhornyak/SCXML-tutorial/blob/master/Tests/ecma/W3C/Mandatory/Auto/test364.scxml
in SCXML Editor and run it, I receive:
ERROR> 2023-Jun-18 21:03:14.750596 uscxml_tester.exe Issue (FATAL) at //state[@id="s2"]/initial[1]: Initial element must define exactly one transition

**Comment by alexzhornyak** [comment, marked as answer]
Hi, [uscxml](https://github.com/tklab-tud/uscxml) has an option of static scxml analysis before execution. Sometimes it may give not correct results. In this case it is a mistake and state chart is a valid, because one transition target from initial element must be in non-parallel states, but here we have targets as the descendants of parallel state, so it's ok

So, you can disable static analysis in the ScxmlEditor settings to pass the test.

![image](https://github.com/alexzhornyak/ScxmlEditor-Tutorial/assets/18611095/b0173faa-bfba-4c03-bfad-7c82762b27a9)
